### PR TITLE
Define a function prototype for YieldProcessor

### PR DESCRIPTION
**Pull request by jkoritzinsky**
Fixes official build failures.

### CODE REVIEW

**Review by steveisok — APPROVED** [review]

File: src/native/minipal/time.c [hunk 1]
```diff
@@ -30,6 +30,7 @@ int64_t minipal_hires_tick_frequency()
 #include <time.h> // nanosleep
 #include <errno.h>
 
+inline static void YieldProcessor();
 inline static void YieldProcessor()
```
Comment (am11) [inline src/native/minipal/time.c:34]:
```suggestion
inline static void YieldProcessor(void)
```
?

**Comment by am11** [comment]
I think minipal deserves its own label. 🏷️

File: src/native/minipal/time.c [hunk 2]
```diff
@@ -30,7 +30,7 @@ int64_t minipal_hires_tick_frequency()
 #include <time.h> // nanosleep
 #include <errno.h>
 
-inline static void YieldProcessor()
+inline static void YieldProcessor(void)
```
Comment (jkoritzinsky) [inline src/native/minipal/time.c:33]:
```suggestion
inline static void YieldProcessor(void);

inline static void YieldProcessor(void)

```

**Comment by am11** [comment]
It must be due to `inline` otherwise none of the helpers required separate prototype https://github.com/dotnet/runtime/blob/6c7ece9479c7cbb0ca6e372adf0a0154a5f0c1a0/src/native/minipal/utf8.c. We can simply drop the `inline` and let compiler decide (`inline` is just a hint and compiler can choose to ignore it if present and vice versa).

**Comment by jkoritzinsky** [comment]
Honestly I just want to get the simple change in to fix the official build. We can do more cleanup in the future.

**Comment by am11** [comment]
I didn't noticed official build mention. Good call!

**Pull request merged**